.github/CONTRIBUTING.md

```diff
@@ -149,14 +147,13 @@ Dredd is tested on the [AppVeyor][], a Windows-based CI. There are still [severa
 
 ### Linting
 
-Dredd uses [coffeelint][] to lint the CoffeeScript codebase. There is a plan
-to converge with Apiary's [CoffeeScript Style Guide][], but as most of
-the current code was written before the style guide was introduced, it's
-a long run. The effective settings are in the [coffeelint.json][] file.
+Dredd uses [eslint][] to lint the JavaScript codebase. We are using Airbnb's
+styleguide rules as a baseline with several rules disabled to fit our own
```

Comment:
styleguide link?

Reply:
and the rules are disabled not to fit our own preferences, AFAIK, but to allow us to have dirty post-decaffeinate code which we're going to gradually improve until one day, all overridden rules are gone 🤤